### ISSUE TITLE
Exclusion List Production Release Date?  Follow Up From Issue #210 

### ISSUE THREAD

**Issue by jmetz272003** [AdguardTeam/HttpsExclusions#226]
Hello, We owe our client of Franklin Templeton an update hence asking for update here.  We were informed this would be completed in 2-3 weeks from October 23rd, and merely looking for confirmation the exclusion list migrated to production.  

Earlier Notes from earlier Issues:  
Has a date been derived for exclusion list to migrate to production?
Last update was in Issue 205 and previously 191, and adding comments to closed issues hasn't netted any replies hence opening a new issue.

Thank You,
 Jeff

From: zzebrum notifications@github.com
 Sent: Thursday, October 10, 2019 11:09 AM
 To: AdguardTeam/HttpsExclusions HttpsExclusions@noreply.github.com
 Cc: Metz, Jeff P. jeffrey.metz@franklintempleton.com; Mention mention@noreply.github.com
 Subject: Re: [AdguardTeam/HttpsExclusions] Release Date for AdGuard for Mac version 2.2? (#205)

@jmetz272003 hello there! Currently, we are preparing for the first beta. I'll try to manage to update the exclusions list earlier.
—
You are receiving this because you were mentioned.
 Reply to this email directly, view it on GitHub, or unsubscribe

ameshkov commented Oct 24, 2019  
Hi @jmetz272003 and sorry for the late reply!

The current ETA for the v2.2 release is 2-3 weeks from now. We've almost finished everything we planned.

**Comment by jmetz272003** [comment]
Hi Adguard support,  Any update if this exclusion is implemented in production?  
Thank you.

**Comment by jmetz272003** [comment]
Hi Adguard Support,  Any update by chance?  Thank you.

**Comment by ameshkov** [comment]
@jmetz272003 hi! We've released the new version, but as I see we failed to update the HTTPS exclusions list, I am terribly sorry for this.

On the bright side, we can solve this issue without the app update and I don't really understand why haven't we done it in the very beginning.

Here's the change that completely disables ALL filtering for the IP addresses of the `profile.franklintempleton.com` domain: https://github.com/AdguardTeam/AdguardFilters/commit/ef56a1561dfa0e03c5179cfadfee4b4f0032ffa6

It will take a couple of hours for AdGuard filter lists to be updated, but once they are, the issue will go away.

I apologize that it took so long to make this change, I simply forgot that we had this option.
